Description: Remove OpenSSL references from kQOAuth
 The code will happily build without OpenSSL.
Author: James Cowgill <jcowgill@debian.org>
Forwarded: not-needed
Justification: CLA not signed

--- a/thirdparty/kQOAuth/CMakeLists.txt
+++ b/thirdparty/kQOAuth/CMakeLists.txt
@@ -46,7 +46,7 @@ if (NOT MSVC)
    set_target_properties (
       kqoauth
       PROPERTIES
-         COMPILE_FLAGS "${PCH_INCLUDE} -I ${PROJECT_SOURCE_DIR}/thirdparty/openssl/include -g -Wall -Wextra -Winvalid-pch"
+         COMPILE_FLAGS "${PCH_INCLUDE} -g -Wall -Wextra -Winvalid-pch"
       )
 else (NOT MSVC)
    set_target_properties (
--- a/thirdparty/kQOAuth/kqoauthutils.cpp
+++ b/thirdparty/kQOAuth/kqoauthutils.cpp
@@ -24,11 +24,6 @@
 #include <QtDebug>
 #include "kqoauthutils.h"
 
-#include <openssl/pem.h>
-#include <openssl/err.h>
-#include <openssl/ssl.h>
-#include <openssl/evp.h>
-
 
 QString KQOAuthUtils::hmac_sha1(const QString &message, const QString &key)
 {
